cli/internal/reference/compref/compref.go

```diff
@@ -39,6 +39,9 @@ var (
 // DefaultPrefix is the default prefix used for component descriptors.
 const DefaultPrefix = "component-descriptors"
 
+// CtfArchiveExtensions is the list of archive file extensions that should be treated as CTF
+var CtfArchiveExtensions = [...]string{".tar.gz", ".tgz", ".tar"}
```

Comment:
```suggestion
var CtfArchiveExtensions = [...]string{".tar.gz", ".tgz", ".tar"}
```

why make this a fixed length array if its a public var? I suggest you either make it package private or leave it as slice for extensions by users.

Reply:
I think the reference parser is opinionated towards to CTF, OCI and CLI. I think this is fine for that reason